### PR TITLE
Do not throw 500 on client connection close

### PR DESCRIPTION
**Pull request by adyach**
# One-line summary
Do not throw 500 on client connection close, during data transmit

### CODE REVIEW

**Comment by adyach** [comment]
👍

File: src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java [hunk 1]
```diff
@@ -84,6 +86,11 @@ private String generateErrorTraceId() {
         return create(Problem.valueOf(BAD_REQUEST, message), request);
     }
 
+    @ExceptionHandler(EofException.class)
+    public void handleEofException() {
+        LOG.info("Client closed connection");
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java]: I don't really see much value in this log as it doesn't provide any specific information like subscription id or session id. Is it possible to have this info?

**Comment by v-stepanov** [comment]
👍

File: src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java (hunk 1 above)
Comment (v-stepanov) [reply src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java]: Ok, you pointed out that FlowId will be there. Good for me.

**Pull request merged**